thirdparty/assimp/include/assimp/types.h

```diff
@@ -161,6 +161,13 @@ struct aiColor3D
     explicit aiColor3D (ai_real _r) : r(_r), g(_r), b(_r) {}
     aiColor3D (const aiColor3D& o) : r(o.r), g(o.g), b(o.b) {}
 
+    aiColor3D& operator= (const aiColor3D& o) {
```

Comment:
This should be contributed upstream in priority: https://github.com/assimp/assimp

Reply:
Yep, I've just done that:
https://github.com/assimp/assimp/pull/2528